### ISSUE TITLE
Switching between template files inside custom_dir for different languages

### ISSUE THREAD

**Issue by Kanaduchi** [ultrabug/mkdocs-static-i18n#49]
I have a case when in custom_dir of mkdocs theme I have some overrides of template html files (main.html, home.html):
```
theme:
  name: material
  custom_dir: overrides
  favicon: assets/icon.png
...
```

Is it possible to add possibility of switching between template files inside this dir for different languages?

Content of custom_dir folder:
- overrides/
  |_ main.html
  |_ main.fr.html

**Comment by ultrabug** [comment]
Hi @Kanaduchi 

That's a good question indeed. I never checked about where those files are taken into account.

I'll have a look and see what's doable.

**Comment by ultrabug** [comment]
As I suspected the `custom_dir` parameters is injected to Jinja's environment from the theme loading logic, which means that babel's translation logic should be used to localize theme related texts.

As for now, this plugin only focused on localizing the **pages** content of your site, not the theme itself.

We worked on documenting how theme developers can localize their theme here : https://www.mkdocs.org/dev-guide/translations/

Hope this helps.
